internal/apps/msp/apm/notifygroup/notifygroup.service.go

```diff
@@ -84,8 +82,7 @@ func (n *notifyGroupService) CreateNotifyGroup(ctx context.Context, request *pb.
 	if err != nil {
 		return nil, errors.NewInternalServerError(err)
 	}
-	c := utils.NewContextWithHeader(ctx)
-	c = transport.WithHeader(c, metadata.New(map[string]string{httputil.InternalHeader: "true"}))
+	c := apis.WithInternalClientContext(ctx, discover.MSP())
```

Comment:
`discover.MSP()` will return the address of msp.
You should use `discover.SvcMSP` instead.